packages/components-js/src/pages/text-field-wrapper.html

```diff
@@ -195,6 +195,29 @@
         value="some really long password with many words and amazing special characters, letters big and small, numbers"
```

Comment:
The value text of "Password" is not correct (also on other `type="search"` examples).